lib/readwise/cli/document/create_command.rb

```diff
@@ -0,0 +1,127 @@
+require_relative '../base_command'
+
+module Readwise
+  class CLI
+    module Document
+      class CreateCommand < BaseCommand
+        def banner
+          "Usage: readwise document create [options]"
+        end
+
+        def description
+          "Sends HTML content to Readwise Reader API"
+        end
+
+        def add_options(opts)
+          opts.on("-f", "--html-file=FILE", "HTML file path") do |file|
+            options[:file] = file
+          end
+
+          opts.on("--title=TITLE", "Document title") do |title|
+            options[:title] = title
+          end
+
+          opts.on("--author=AUTHOR", "Document author") do |author|
+            options[:author] = author
+          end
+
+          opts.on("-u", "--url=URL", "Source URL (defaults to https://example.com/<filename>)") do |url|
+            options[:url] = url
+          end
+
+          opts.on("--summary=SUMMARY", "Document summary") do |summary|
+            options[:summary] = summary
+          end
+
+          opts.on("--notes=NOTES", "Personal notes") do |notes|
+            options[:notes] = notes
+          end
+
+          opts.on("--location=LOCATION", "Document location: new, later, archive, feed (default: new)") do |location|
+            unless %w[new later archive feed].include?(location)
+              puts "Error: Invalid location. Must be one of: new, later, archive, feed"
+              exit 1
+            end
+            options[:location] = location
+          end
+
+          opts.on("--category=CATEGORY", "Document category: article, email, rss, highlight, note, pdf, epub, tweet, video") do |category|
+            unless %w[article email rss highlight note pdf epub tweet video].include?(category)
+              puts "Error: Invalid category. Must be one of: article, email, rss, highlight, note, pdf, epub, tweet, video"
```

Comment:
Can we extract out `CATEGORIES = %w[article email rss highlight note pdf epub tweet video]` and then reference that in these three places?